portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/greeting")
+public class GreetingServlet extends HttpServlet {
+    
+    /*
```

Comment:
In general, you should avoid checking in commented out code. The point of version control (e.g. Git) is that you can just delete code that you (previously) checked in and get it back.

Reply:
Let me be clearer. Remove the commented out code.